ui/src/shared/components/MultiSelectDropdown.js

```diff
@@ -39,14 +39,6 @@ class MultiSelectDropdown extends Component {
     this.setState({isOpen: false})
   }
 
-  componentWillReceiveProps(nextProps) {
```

Comment:
this was added in September as part of a commit introducing TICKScript Editor update functionality. i think simply removing it, without understanding why it was introduced, would be too blunt. how about introducing a prop to toggle this check, that is only set to true within the context where you are trying to stop the offending behavior?